README.md

```diff
@@ -16,6 +16,20 @@ gem 'iex-ruby-client'
 
 Run `bundle install`.
 
+
+## Methods Available:
+
+- [Get a Single Price](#get-a-single-price)
+- [Get a Quote](#get-a-quote)
+- [Get a OHLC (Open, High, Low, Close) price](#get-a-ohlc-open-high-low-close-price)
+- [Get a Market OHLC (Open, High, Low, Close) prices](#get-a-market-ohlc-open-high-low-close-prices)
+- [Get Company Information](#get-company-information)
+- [Get Company Logo](#get-company-logo)
+- [Get Recent News](#get-recent-news)
+- [Get Chart](#get-chat)
+- [Get Key Stats](#get-key-stats)
```

Comment:
This is OK here, but consider adding a TOC with [danger-toc](https://github.com/dblock/danger-toc) instead.